### ISSUE TITLE
Change color saturation per application

### ISSUE THREAD

**Issue by ErikReider** [WillPower3309/swayfx#20]
### Please fill out the following:
- **Description:**
Please describe in plain English what the enhancement is and what the use case is.

To be able to change the color saturation of a container through the config. This will make it easier to see enemies in games like CSGO or any other competitive game.

Example:
```
# Just like the opacity cmd
for_window [class="csgo_linux64"] saturation set 2.0
```

**Comment by WillPower3309** [comment]
Great work implementing this! One thing to bear in mind is that this may be something semi supported in mainline sway soon(ish):

https://gitlab.freedesktop.org/wayland/wayland-protocols/-/merge_requests/14

The protocol specifically has a saturation field too! I haven't looked into how it will work per client vs the entire output but something to keep an eye on. This could be great to include leading up to that protocol being merged however (especially considering how long it's been in progress)
